app/assets/javascripts/mask.js

```diff
@@ -0,0 +1,16 @@
+window.onload = function () {
+    Array.from(document.getElementsByClassName('phone-input')).forEach(function (element) {
+        new IMask(element, {
+            mask: '+{38}(000)000-00-00',
+            lazy: false,
+        });
+    });
+};
+
+function changePhone() {
```

Comment:
Use better name for this function: unmaskPhone

Reply:
@VasylYaremkiv change